scss/_reboot.scss

```diff
@@ -248,11 +248,11 @@ sup { top: -.5em; }
 // Links
 
 a {
-  color: $link-color;
+  color: var(--#{$variable-prefix}link-color, $link-color);
   text-decoration: $link-decoration;
 
   &:hover {
```

Comment:
@mdo Is there any reason we didn't double with `:focus`, here? It'd make focused link stand out a bit more, and would unveil simpler colored link overrides.

Reply:
`:active` is when an element is being activated by a user - the time between when a user presses a mouse button and then releases it. https://stackoverflow.com/a/1678006/2453148

I think we do not have an `:active` state without a `:hover` state.

Reply:
Do you suggest we also add an `:active` state alongside `:focus` and `:hover`?

Reply:
@ffoodd While I was testing, I forgot what I was testing. As you wish, is't is not critial i think.

Reply:
Indeed, I'd only go for `:focus` for now 😄